Add a widnows section in SSI docs (#29395)
DataDog/documentation: content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm/windows.md

---
title: Single Step APM Instrumentation on Windows
code_lang: windows
type: multi-code-lang
code_lang_weight: 30
further_reading:
  - link: /tracing/metrics/runtime_metrics/
    tag: Documentation
    text: Enable Runtime Metrics
---

> <div class="alert alert-info"> Single Step APM Instrumentation on Windows is currently in preview.</div>

Reviewer: ```suggestion
<div class="alert alert-info"> Single Step APM Instrumentation on Windows is in Preview.</div>
```

## Overview

With Single Step Instrumentation (SSI), you can automatically enable APM for your .NET Framework and .NET Core applications running on Windows IIS, using a single MSI command during the Datadog Agent installation. No separate SDK installation or code changes are required.

## Enable APM on Windows

To enable APM with Single Step Instrumentation on Windows:

> 1. In the Datadog app, go to the Install Datadog [Windows Agent page][1] .

Reviewer: ```suggestion
1. In Datadog, go to the Install Datadog [Windows Agent page][1] .
```

> 1. Under Customize your observability coverage, toggle APM Instrumentation on.

Reviewer: suggesting to use the exact words in the UI

```suggestion
1. In the **Customize your observability coverage** section, toggle **Application Performance Monitoring (APM)**.
```

> 1. Copy and run the provided MSI install command on your Windows host.

Reviewer: not sure if this is important enough to mention. if not, disregard.

1. Restart the IIS applications you want instrumented. (You do not need to restart the entire IIS server.)

> Once installed, the Agent automatically loads the Datadog .NET SDK into supported application processes to enable distributed tracing.

Reviewer: ```suggestion
After installation, the Agent automatically loads the Datadog .NET SDK into supported application processes to enable distributed tracing.
```

> **Note:** Only .NET applications running in IIS are supported for Single Step Instrumentation on Windows.

Reviewer: this seems like an important note to mention upfront. can we move it up to just after the h2?

## Set SDK Tracer Versions

By default, Single Step Instrumentation installs the latest supported version of the Datadog .NET Tracer.

If you need to pin a specific version:

> 1. From the Windows Agent install page, after toggling APM Instrumentation, select Customize library versions.

Reviewer: ```suggestion
1. From the Windows Agent install page, after toggling **Application Performance Monitoring (APM)** instrumentation, select Customize library versions.
```

1. Under .NET, choose the version you want to use.
1. Copy the updated MSI command and run it on your host.

## Remove Single Step APM instrumentation from your Agent

> To disable SSI for .NET on your host run:

Reviewer: ```suggestion
To disable SSI for .NET on your host, run:
```
